Cargo.toml

```diff
@@ -1,14 +1,14 @@
 [package]
 name = "membarrier"
-version = "0.1.0"
+version = "0.2.0"
 authors = ["Jeehoon Kang <jeehoon.kang@sf.snu.ac.kr>"]
 license = "MIT/Apache-2.0"
 readme = "README.md"
 repository = "https://github.com/jeehoonkang/membarrier-rs"
 homepage = "https://github.com/jeehoonkang/membarrier-rs"
 documentation = "https://docs.rs/membarrier"
 description = "Memory barrier"
-keywords = ["linux", "membarrier"]
+keywords = ["process-wide memory barrier", "membarrier", "FlushProcessWriteBuffers"]
```

Comment:
Maybe also add "sys_membarrier" and "rcu"?

I'm not sure if "process-wide memory barrier" is a good keyword, but it would be a better description than "Memory barrier". :)